### PR TITLE
London Class 8 - Amarachi Okereke - JavaScript-Core-1-Coursework-Week 1

### PR DESCRIPTION
**Pull request by AmarachiOkereke**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name: Amarachi Okereke
- Your City: London
- Your Slack Name: Amarachi Okereke

# Homework Details

- Module: JavaScript-Core-1-Coursework
- Week: 1

# Notes

- What did you find easy?
 Writing functions that add, multiply or that takes a string as an input.

- What did you find hard?
 Manipulating functions like calculating percentages, tax etc. It's a bit confusing 

- What do you still not understand?
 Some questions relating to functions are still a challenge

- Any other notes?
 I will like to request a follow up teaching on functions please if possible to help me have a better understanding of function.

### CODE REVIEW

**Review by DelroyGayle — COMMENTED** [review]
Dear Amarachi,
I suggest that all your functions should be placed at the top of the program then followed by the code. That is, the definitions of function add(a, b) {} ... function multiply(a, b) ... function format(number) etc should be at the top; then followed by let's, console.log's etc. 
So definitions of functions first, followed by the code that uses them. That generally, is the structured approach to writing programs.

File: mandatory/1-syntax-errors.js [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
 
 function getTotal(a, b) {
-  total = a ++ b;
+  return "The total is " + (a + b);
 
-  return "The total is total";
+  
 }
-
+var result = getTotal(14, 14);
```
Comment (DelroyGayle) [inline mandatory/1-syntax-errors.js:18]: 'var' is now considered outdated. Please use 'let' instead

Comment (AmarachiOkereke) [reply mandatory/1-syntax-errors.js]: Yeah, I have corrected it.

**Comment by AmarachiOkereke** [comment]
Thank you Delroy for your review. Can you please indicate where functions are not correctly written in my code, I have checked my code and it seems to be in the correct order. Thank you

> Dear Amarachi, I suggest that all your functions should be placed at the top of the program then followed by the code. That is, the definitions of function add(a, b) {} ... function multiply(a, b) ... function format(number) etc should be at the top; then followed by let's, console.log's etc. So definitions of functions first, followed by the code that uses them. That generally, is the structured approach to writing programs.

**Pull request closed without merging**